### ISSUE TITLE
selecting east/west for ographic definitions

### ISSUE THREAD

**Issue by thareUSGS** [PlanetMap/csvForWKT#6]
I think with the recent changes the only item to tackle is setting east/west for ographic definitions. 

I think this should catch all variants.

if (earth or moon)  # always east
    lonDirection = east
else if (ographic AND rotation=Retrograde) then
   lonDirection = west
else
   lonDirection = east

**Comment by J-Christophe** [comment]
I do not understand the problem. Is there a problem on a WKT ? For example ?

If I look to your algo, it seems a case is missing for ographic definitions. 

There are three cases for "ographic" definitions:

- Historical reasons : Sun, Earth, Moon => longitude positive to East

- Planets and their satellites : when rotation is Direct then longitudeDirection is positive to west (north pole is relative to invariant plane)

- dwarf planets, minor planets, their satellites and comets : longitudeDirection is positive to east (always) (north pole is relative to positive spin pole. So the rotation is always Direct (p21 from IAU document). After reading the whole page 21 again, It seems the axis should be called longitude/latitude planetocentric/planetodetic for elliposid reference surface. And what about triaxial axis ?

**Comment by thareUSGS** [comment]
Yes and the Sun. I must have been looking at the wrong mapping number, west looks to be used correctly. :+1:

Thanks for pointing out page 21-22, _"..., if referenced to the body center, latitude and longitude on such bodies may be called planetocentric, and planetodetic if referenced to an ellipsoidal reference surface."_

Now try to get thought these slides 23-25. Huh
https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/Tutorials/pdf/individual_docs/17_frames_and_coordinate_systems.pdf#page24

slide 24: _"Planetodetic longitude is the same as planetocentric longitude"_
slide 25: _"Planetographic **latitude** is planetodetic **latitude**"_

I will try to get some additional clarification, but I think for dwarf planets, minor... we can keep calling it planetographic for ellipses (since they are the same).

Perhaps we can use the same for triaxial but I will also check on that too.
